### PR TITLE
[dev] Monorepo: drop .wireit cache folders older than four weeks

### PR DESCRIPTION
**Pull request by kalessil**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Context: monorepo feedback from eng. teams

- Drop stale cache directories when pulling changes from origin

### How to test the changes in this Pull Request:

- Execute the new commands locally; they should succeed

### CODE REVIEW

**Review by dinhtungdu — APPROVED** [review]
Thank you so much for addressing this issue so quickly. I left an optional comment but the change is LGTM! I tested the script manually on macOS and it successfully removed the cache. I also tested with Alpine Linux v3.16 and the command works there too. I don't have a Windows machine so I can't test, but I think it'd work with WSL.

**Comment by kalessil** [comment]
@dinhtungdu: thanks!

> I left an optional comment

Hm, I don't see it

> I don't have a Windows machine so I can't test, but I think it'd work with WSL.

All good; if commands work on MacOS and Linux, they will also work in WSL.

File: .husky/post-merge [hunk 1]
```diff
@@ -3,10 +3,16 @@
...
 	printf "It was a change in the following file(s) - refreshing dependencies:\n"
 	printf "    %s\n" $changedManifests
 
 	pnpm install --frozen-lockfile
 fi
+
+# Cleanup .wireit cache that is older than 4 weeks; otherwise, the repository directory size keeps growing and growing.
+find plugins/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
+find packages/js/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
+find plugins/woocommerce/client/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
```
Comment (dinhtungdu) [inline .husky/post-merge]: Should we leave some messages here, at the start and end of the cache removal script explaining what is happening?

Comment (dinhtungdu) [reply .husky/post-merge]: @kalessil here is the comment, looks like I didn't submit it 😅

Comment (kalessil) [reply .husky/post-merge]: Makes sense: pushing ceee320beb60e3f36c0f392322ea675193d92c3e, which prompts about the cache cleanup (prompt before cleanup should be enough - don't want to overload console output).

Comment (kalessil) [reply .husky/post-merge]: I'll re-test the changes on MacOS/Linux and merge afterwards.

Comment (dinhtungdu) [reply .husky/post-merge]: Thanks! I tested the updated scripts manually and it works great on my end.

**Pull request merged**